src/components/github-contributors-list/github-contributors-utils.ts

```diff
@@ -0,0 +1,39 @@
+export type GithubContributor = {
+  name: string;
+  href: string;
+  key?: string;
+  contributions?: number;
+};
+
+export type GithubContributorResponse = {
+  id: number;
+  login: string;
+  html_url: string;
+  contributions: number;
+};
```

Comment:
WDYT about using a different file for types? They are exported so they are not relevant only for utils